src/includes/performance/set-transaction-current-scope/java.mdx

```diff
@@ -35,7 +35,7 @@ import io.sentry.Sentry
 
 fun executeTask() {
   val transaction = Sentry.startTransaction("transaction-name", "task")
-  Sentry.configureScope(scope -> scope.setTransaction(transaction))
+  Sentry.configureScope { it.transaction = transaction }
```

Comment:
enriching-events/scopes/with-scope/java.mdx changed to using a `setTransaction` instead of `transaction = transaction`, is that correct? or is it different because of the use in `configureScope` and `withScope`?

Reply:
In Kotlin, `setTransaction` takes only `String`, `transaction = ` takes `ITransaction`. Perhaps we can make it more explicit, change `setTransaction(String)` into `setTransactionName` since it's a breaking change.

Reply:
oh yeah true, they are not the same property